templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -120,6 +120,78 @@
 				</form>
 			</div>
 		</div>
+		{{if .Repository.IsTimetrackerEnabled }}
+		{{if .CanUseTimetracker }}
+		<div class="ui divider"></div>
+		<div class="ui timetrack">
+			<span class="text"><strong>{{.i18n.Tr "repo.issues.tracker"}}</strong></span>
+			<div>
+						<form method="POST" action="{{$.RepoLink}}/issues/{{.Issue.Index}}/times/stopwatch/toggle" id="toggle_stopwatch_form">
+							{{$.CsrfTokenHtml}}
+						</form>
+						<form method="POST" action="{{$.RepoLink}}/issues/{{.Issue.Index}}/times/stopwatch/cancel" id="cancel_stopwatch_form">
+							{{$.CsrfTokenHtml}}
+						</form>
+						{{if  $.IsStopwatchRunning}}
+							<div class="ui buttons fluid">
+								<button onclick="toggleStopwatch()" class="ui button">{{.i18n.Tr "repo.issues.stop_tracking"}}</button>
+								<button onclick="cancelStopwatch()" class="ui negative button">{{.i18n.Tr "repo.issues.cancel_tracking"}}</button>
+							</div>
+						{{else}}
+							{{if .HasUserAStopwatch}}
+								<div class="ui warning message">
+									{{.i18n.Tr "repo.issues.tracking_already_started" .OtherStopwatchURL | Safe}}
+								</div>
+							{{end}}
+							<div class="ui buttons two fluid">
+								<button onclick="toggleStopwatch()" class="ui button poping up" data-content='{{.i18n.Tr "repo.issues.start_tracking"}}' data-position="top center" data-variation="small inverted">{{.i18n.Tr "repo.issues.start_tracking_short"}}</button>
+								<button onclick="timeAddManual()" class="ui icon button green poping up" data-content='{{.i18n.Tr "repo.issues.add_time"}}' data-position="top center" data-variation="small inverted"><i class="octicon octicon-plus"></i> </button>
```

Comment:
Button should have label `Add` can be with icon `+` but label is needed to be easier to understand what does it mean